README.md

```diff
@@ -1,6 +1,6 @@
 # Create Envfile Github Action
 
-**Name: SpicyPizza/create-envfile@v1**
+**Name: SpicyPizza/create-envfile@master**
```

Comment:
I would much prefer to leave this as versions so that features can be developed on master if needed without functionality changing under the user. But if the docs don't suggest this workflow, feel free to send me some references.

Of course, for your use case you can make use of master, but in general recommending it in the readme isn't a great idea I think.

Reply:
@AngelOnFira I'll revert this change, and when this PR is merged you can create a new release and then you can update readme according to the new version.